src/Graph.ts

```diff
@@ -73,3 +73,17 @@ function isAnExit(board: Board, x: number, y: number) {
     y === lastRowIndex
   );
 }
+
+export const graphToBoard = (graph: Graph): Board => {
+  let board: Board = [[]];
+  let maxDimension = 0;
+  for (const index in graph.nodes) {
+    const node: Node = graph.nodes[index];
+    if (node.y > maxDimension) {
+      maxDimension++;
```

Comment:
I think maxDimension is not necessary.
You can test if board[node.y] is defined and is an array

Reply:
note for myself for further reference : do not overthink something simple :D